src/docs/sdk/envelopes.mdx

```diff
@@ -113,9 +113,9 @@ Envelopes can have a number of headers which are valid in all situations:
   the envelope has all the information necessary to be sent to sentry.  In this
   case the full DSN must be stored in this key.
 
-`sdk_info`
+`sdk`
 
-: *Object, recommended.* This can carry the same payload as the <Link to="/sdk/event-payloads/debugmeta/">`sdk_info` attribute</Link>
+: *Object, recommended.* This can carry the same payload as the <Link to="/sdk/event-payloads/sdk/">`sdk` interface</Link>
   of the `debug_meta` interface in the event payload but can be carried for all events.
```

Comment:
```suggestion
: *Object, recommended.* This can carry the same payload as the <Link to="/sdk/event-payloads/sdk/">`sdk` interface</Link>
  in the event payload but can be carried for all events.
```

Reply:
`of the debug_meta` should be gone